lib/Log/Any/Plugin/Stringify.pm

```diff
@@ -13,7 +13,8 @@ use Data::Dumper;
 sub install {
     my ($class, $adapter_class, %args) = @_;
 
-    my $stringifier = $args{stringifier} || \&default_stringifier;
+    my $sep = defined $args{separator} ? $args{separator} : '';
```

Comment:
If you move `$sep` up into the top-level scope of the module, you don't need all the _builder complications.

Also, seeing as we've already got `$stringifier` as a name, might as well spell out `$separator`.

Reply:
Done!